### DISCUSSION TITLE
Cannot get "Insert Media" to display videos

### DISCUSSION THREAD

**Discussion by TheBigShort111** [requarks/wiki#3332] [Error / Bug Report]
**Describe the bug**
Using the Virtual Editor I cannot get any videos from URL's to show/or play using the "Insert Media" button at the top of the edit page. For example, if I select a youtube URL, a static representation of the video will show on the not yet saved page, indicating that the insert media did find the video via the URL, however, one the page is saved, the video completely disappears. 

**To Reproduce**
1. Go to 'edit page'
2. Click on 'insert media'
3. Select URL for media to display'
4. Click save page
5. Video disappears

**Expected behavior**
After selecting insert media and saving the page I'd expect to see the video I was trying to insert

**Screenshots**
https://www.dropbox.com/s/z4o27pxd7a4ec3m/Screen%20Shot%202021-02-24%20at%2012.45.55%20AM.png?dl=0

Shows in page edit mode (static, video does not play)
https://www.dropbox.com/s/waomtf02vy0ka9r/Screen%20Shot%202021-02-24%20at%2012.46.48%20AM.png?dl=0

Save page and the video completely disappears
https://www.dropbox.com/s/q15vc4nwnlrdhi4/Screen%20Shot%202021-02-24%20at%2012.48.32%20AM.png?dl=0

**Host Info (please complete the following information):**
 - OS: Docker Container (Linux)
 - Wiki.js version: 2.5.170
 - Database engine: PostgreSQL 11.11 db

It is worth noting that I have tried this as an administrator with all permissions, so it is not permission related. Also, I have tried this by:
- Disabling "Block Open Redirect" in security settings - no change
- I have tried enabling Media Players in the HTML Pipeline under Rendering - no change
- I have tried disabling Sanitize HTML in: html/security - no change
- I have enabled allow iframes in html/security - no change
- I have tried various combinations of the above - no change

Again, this issue is when using the "Insert Media" button from within the visual editor

Thanks for any help you can provide

**Comment by TheBigShort111** [comment]
I see there was a PR to resolve this issue, but the web admin of our server says the PR cannot be easily implemented as it is part of Docker that he is not familiar with. Is this a difficult patch? What are the steps involved? - Does the wiki need to be shut down? I'd really like to get video embedding to work through the media embedder in the visual editor - Any insight or clarity on how to accomplish this would be very helpful.

**Comment by Jsmithrud37** [comment]
I'm also running into this. Any help would be greatly appreciated.

**Comment by radialbalance** [comment]
Boosting this thread as well because we really need iFrame support on the Visual Editor in order to move forward with this wiki for our project. We're loving everything about Wiki.js, just need this working and we're set!

**Comment by bonesoul** [comment]
same here

**Comment by JamesHarley** [comment]
I'm also having this issue

**Comment by TimCottee** [comment]
Same problem here - although no docker, running on Windows 2019 with latest postgres etc., was hoping 2.5.191 would fix this but no, sadly not.

**Comment by jaumecornado** [comment]
Another one running on kubernetes & postgres

**Comment by JamesHarley** [comment]
I'm having this issue as well but not in Docker. I'm running it from centos with a manual install.

**Comment by itsmemac** [comment]
Same issue here as well.
Running Docker with Ubuntu 20.04
In edit mode, its all good but when the page is rendered, the video is no where to be seen
Used all the editor modes available to check if its wysiwyg editor issue but it ain't

**Comment by Kaibert123** [comment]
I'm also running into this issue, using the wysiwyg editor - cannot get the Vimeo URL video to appear outside of edit mode.

**Comment by AVTPJ** [comment]
Same here, running a test server (2.5.214) on Windows with Postgre. No comments from devs? Why was the issue closed?

**Comment by Iam1337** [comment]
Hi! Same here, docker, ver 2.5.268. 

@NGPixel Hello, sorry for disturbing, but can you tell something about that issue?
